Today Extension/LoadingTableViewCell.swift

```diff
@@ -0,0 +1,37 @@
+//
+//  LoadingTableViewCell.swift
+//  Today Extension
+//
+//  Created by Yana Sang on 3/6/19.
+//  Copyright © 2019 cuappdev. All rights reserved.
+//
+
+import UIKit
+
+class LoadingTableViewCell: UITableViewCell {
+
+    let loadingIndicator = LoadingIndicator()
+    let verticalMargin: CGFloat = 20.0 // top & bottom margin
+
+    override init(style: UITableViewCell.CellStyle, reuseIdentifier: String?) {
+        super.init(style: style, reuseIdentifier: reuseIdentifier)
+
+        contentView.addSubview(loadingIndicator)
+    }
+
+    override func layoutSubviews() {
+        super.layoutSubviews()
+
+        loadingIndicator.snp.makeConstraints { make in
```

Comment:
should just put these constraints inside the `init` and not use `layoutSubviews()`